cogctl/cli/profile.py

```diff
@@ -42,8 +42,9 @@ def create(state, name, host, port, secure, rest_user, rest_password):
     """
     Add a new profile to a `.cogctl` file.
     """
-    add_profile(state.config_file, name, {"host": host,
-                                          "port": port,
-                                          "secure": secure,
-                                          "user": rest_user,
-                                          "password": rest_password})
+    state.configuration.add(name, {"host": host,
+                                   "port": port,
+                                   "secure": secure,
+                                   "user": rest_user,
+                                   "password": rest_password})
```

Comment:
Wouldn't this add a profile in the "old" style breaking secure, host and port apart?